lib/java/com/google/android/material/slider/Slider.java

```diff
@@ -976,8 +998,11 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
         thumbPosition = position;
         snapThumbPosition();
         ViewUtils.getContentViewOverlay(this).remove(label);
+        onStopTrackingTouch();
         invalidate();
         break;
+      case MotionEvent.ACTION_CANCEL:
```

Comment:
I think this can't happen because we call `requestDisallowInterceptTouchEvent(true)`. Do you see a case where this would be called? If so, do we have to do anything else here that happens in `ACTION_UP`?

Reply:
You're right, I'll remove this section.

Reply:
Done